cirq-google/cirq_google/ops/analog_detune_gates.py

```diff
@@ -15,24 +15,14 @@
 """Define detuning gates for Analog Experiment usage."""
 from __future__ import annotations
 
-from typing import AbstractSet, Any, TYPE_CHECKING, TypeAlias
-
-import sympy
-import tunits as tu
+from typing import AbstractSet, Any, TYPE_CHECKING
 
 import cirq
+from cirq_google.experimental.analog_experiments import symbol_util as su
```

Comment:
I am not sure about the dependency graph here.  I don't think we want to import experimental from a non-experimental file.